src/_posts/security/postmortems/2025-03-11-SPM-2025-001.md

```diff
@@ -0,0 +1,87 @@
+
+The following day, after confirming that no further attack waves were occurring, the public IP pool configuration was restored to its original state.
+
+## Timeline of the incident
+
+All times given are in CET (UTC+1)
+
+| **11:08** | Initial signs of unusual network traffic appear in the `osc-fr1` region. |
+| **11:17** | A DDoS attack targeting public IP `148.253.75.120` triggers alerts, indicating the region is unreachable. |
+| **11:25** | An incident ticket is opened with our infrastructure provider to assess if the issue is on their end and to obtain any relevant insights. |
+| **11:30** | The region is fully reachable again. |
+| **11:35** | Analysis of our metrics indicates that we are facing a DDoS attack. |
+| **11:40** | A new wave of attack begins, again targeting `148.253.75.120`. |
+| **11:44** | The incident is escalated with our infrastructure provider, and we reach out to our Technical Account Manager (TAM) for support. |
+| **11:53** | The region is fully reachable again. |
+| **14:16** | A new attack wave is detected, still targeting `148.253.75.120`. |
+| **14:23** | IP `148.253.75.120` is removed from the public IP pool. |
+| **14:26** | The region is fully reachable again. |
+| **14:41** | IP `148.253.75.120` is moved back in the pool. |
+| **14:46** | IP `148.253.75.120` is still under attack, removing the IP from our pool. |
+| **15:23** | A temporary VPC is provisioned, and the targeted IP is linked to it for traffic offloading and analysis. |
+| **16:28** | A new attack wave occurs, this time targeting public IP `109.232.233.130`. |
+| **16:34** | Redirecting traffic for this IP to the previously created isolated VPC. |
+| **16:36** | The platform is fully operational. |
+| **16:45** | A new public IP is added to the pool. |
+| **16:59** | An additional public IP is added to the pool. |
+| **18:32** | Traffic routing to the previously excluded IPs is restored. |
+
+## Impact
+
+- On `osc-fr1`, the cumulative impact on the osc-fr1 region amounts to 49 minutes of unavailability over the course of March 11th.
+
+- On `osc-secnum-fr1`, no impact for our customers.
+
+## Immediate Actions Taken
+
+- The targeted public IP addresses were isolated into a dedicated VPC to preserve platform stability and allow for detailed analysis of the attack traffic.
+- The attacked IP addresses were replaced with new public IPs that were unknown to the attackers, in order to restore service resilience and mitigate further disruption.
+- Observability tools were improved to enable faster identification of the targeted infrastructure components in the event of future attacks. A corresponding incident response procedure was also established to allow for quicker mitigation in similar scenarios.
```

Comment:
Also something like: Internal network management tools have been relocated to an isolated network to safeguard our response capabilities in the event of an attack